public void testAbstractFailure() throws Exception
